packages/ecs-agent/ecs-agent.spec

```diff
@@ -339,44 +192,38 @@ ln -rs %{buildroot}%{_cross_localstatedir}/ecs/managed-agents/execute-command/co
 install -d %{buildroot}%{ecs_exec_dir}/certs
 ln -rs %{buildroot}%{_cross_sysconfdir}/pki/tls/certs/ca-bundle.crt %{buildroot}%{ecs_exec_dir}/certs/tls-ca-bundle.pem
 
-# Prepare license and vendor information so it can be co-installable
-mv %{ecscni_gorepo}-%{ecscni_gitrev}/LICENSE %{ecscni_gorepo}-%{ecscni_gitrev}/LICENSE.%{ecscni_gorepo}
-mv %{vpccni_gorepo}-%{vpccni_gitrev}/LICENSE %{vpccni_gorepo}-%{vpccni_gitrev}/LICENSE.%{vpccni_gorepo}
-# Move vendor folder into a single directory so cross_scan_attribution can run once
-mkdir go-vendor
-mv %{agent_gorepo}-%{agent_gover}/agent/vendor go-vendor/%{agent_gorepo}
-mv %{ecscni_gorepo}-%{ecscni_gitrev}/vendor go-vendor/%{ecscni_gorepo}
-mv %{vpccni_gorepo}-%{vpccni_gitrev}/vendor go-vendor/%{vpccni_gorepo}
-%cross_scan_attribution go-vendor go-vendor
+install -d %{buildroot}{%{_cross_libexecdir},%{_cross_fips_libexecdir}}/amazon-ecs-agent
+# Create symlinks to ECS CNI plugin binaries
+for p in \
+  ecs-bridge \
+  ecs-eni \
+  ecs-ipam \
+; do
+  ln -rs %{buildroot}%{_cross_libexecdir}/cni/ecs/${p} %{buildroot}%{_cross_libexecdir}/amazon-ecs-agent/${p}
```

Comment:
I think in this case we want the symlink to be relative:

```
ln -rs ../../cni/ecs/${p}
```

Reply:
Discussed offline, the `r` flag for `ln` command already makes it relative. We prefer that over:
```
ln -s ../cni/ecs/${p} %{buildroot}%{_cross_libexecdir}/amazon-ecs-agent/${p}
```